### ISSUE TITLE
Hide pagination not working

### ISSUE THREAD

**Issue by viral-prm** [bhaveshpatel200/vue3-datatable#21]
```
 <Vue3Datatable :columns="Jobfields" :rows="lstScheduledJob" :pagination="false" >
</Vue3Datatable>   
```

trying to hide pagination but its not working

**Comment by bhaveshpatel200** [comment]
Hi,

I got it bug and will be solved in next update. meanwhile you can use like below which will works for you.

```
<Vue3Datatable v-if="lstScheduledJob?.length" :columns="Jobfields" :rows="lstScheduledJob" :pagination="false" >
</Vue3Datatable>
```

**Comment by bhaveshpatel200** [comment]
Hi viral-prm,

Above issue has been solved in new version 1.1.4.
